tx.go

```diff
@@ -282,6 +284,8 @@ func (tx *Tx) Commit() (err error) {
 		}
 	}
 
+	tx.db.commitBuffer.Reset()
```

Comment:
It can not use the Reset method here. because it will release the memory want to hold on.

Reply:
According to the comments, Reset will not release the memory, but just clear all the contents inside.The unit test case also tests this case.